packages/react-icons/package.json

```
@@ -35,8 +36,14 @@
     "typescript": "~4.1.0",
     "yargs": "^14.0.0"
   },
+  "peerDependencies": {
+    "@fluentui/react-make-styles": "9.0.0-alpha.70"
```

```suggestion
    "@fluentui/react-make-styles": "^9.0.0-alpha.70"
```

This should still use caret otherwise it will be the same problem as with `dependencies`